scripts/docker-install-dev.sh

```diff
@@ -30,7 +38,8 @@ while test $# -gt 0; do
   esac
 done
 
+
 depends docker
 depends docker-compose
 
-docker-compose build
+docker-compose -f docker-compose-dev.yml build --build-arg DOWNLOAD=${download}
```

Comment:
I made some edits to this file to make it more parametrizable, can you check them out?